### PR TITLE
Install go tools globally on builder

### PR DESCRIPTION
**Pull request by krdln**
This moves installation of go-msi, ginkgo and dep to ansible role
making these tools available globally on builder.

Note: Ansible script is still WIP as it contains at least one bug (robocopy src uses file path instead of directory) but we're going to merge anyway, to make builds stop failing on go-msi issue. The fix for ansible will come in next PR.

### CODE REVIEW

File: ansible/roles/builder/tasks/main.yml [hunk 1]
```diff
@@ -170,6 +170,27 @@
     version: 1.8.3
     state: present
 
+- name: Install go-msi
+  win_chocolatey:
+    name: go-msi
+    state: present
```
Comment (wurbanski) [inline ansible/roles/builder/tasks/main.yml]: Required version should be specified when using public chocolatey.

**Review by wurbanski — CHANGES_REQUESTED** [review]

**Review by wurbanski — DISMISSED** [review]

File: ansible/roles/builder/tasks/main.yml [hunk 2]
```diff
@@ -170,6 +170,28 @@
...
 
+- name: Install go-msi
+  win_chocolatey:
+    name: go-msi
+    version: '1.0.2'
+    state: present
+
+- name: Install dep
+  win_get_url:
+    url: 'https://github.com/golang/dep/releases/download/v0.3.2/dep-windows-amd64'
```
Comment (sodar) [inline ansible/roles/builder/tasks/main.yml:181]: I think that, if we download it like this, then `dep` will be available in `PATH` as `dep-windows-amd64` and not as `dep`.

**Review by sodar — CHANGES_REQUESTED** [review]

File: ansible/roles/builder/tasks/main.yml (hunk 2 above)
Comment (krdln) [reply ansible/roles/builder/tasks/main.yml:181]: Good point

**Review by sodar — DISMISSED** [review]

**Review by sodar — APPROVED** [review]

**Review by kaweue — APPROVED** [review]

**Comment by krdln** [comment]
Ansible script is still WIP as it contains at least one bug (robocopy src uses file path instead of directory) but we're going to merge anyway, to make builds stop failing on go-msi issue. The fix for ansible will come in next PR.

**Pull request merged**